airflow/Dockerfile

```diff
@@ -1,8 +1,5 @@
-# syntax=docker/dockerfile:1@sha256:ac85f380a63b13dfcefa89046420e1781752bab202122f8f50032edf31be0021
+#syntax=docker/dockerfile:1@sha256:ac85f380a63b13dfcefa89046420e1781752bab202122f8f50032edf31be0021
 ARG GIT_SYNC
```

Comment:
I'm curious: git-sync does not need special handling for multi arch?

Reply:
It does, we gonna solve the problem by having a manifest list behind the tag in Harbor. Anyway, I thought that's already the case but it wasn't. Thank you for pointing at it again! I gonna resolve the conversation when this is done